# HG changeset patch
# User Rik <rik@octave.org>
# Date 1502388034 25200
#      Thu Aug 10 11:00:34 2017 -0700
# Node ID 02fbed0e87d5a7f78907beec461810c8f01b96d2
# Parent  4f68899f9e5bd2d6ad9b132e9c10f6e16466a8ec
hdl2struct.m: Fix typo creating unused variable (bug #51722).

* hdl2struct.m: Rename variable "idx2" to "idx".

diff --git a/scripts/plot/util/hdl2struct.m b/scripts/plot/util/hdl2struct.m
--- a/scripts/plot/util/hdl2struct.m
+++ b/scripts/plot/util/hdl2struct.m
@@ -93,17 +93,17 @@ function s = hdl2struct (h)
         s.children(ii) = hdl2struct (lg);
       elseif (nlg > 1)
         error ("hdl2struct: more than one legend found");
       endif
 
       cb = findobj (par, "-depth", 1, "tag", "colorbar");
       if (! isempty (cb))
         ## identify colorbars which are attached to this axes.
-        idx2 = ([get(cb).axes] == h);
+        idx = ([get(cb).axes] == h);
         cb = cb(idx);
       endif
       ncb = length (cb);
       if (ncb == 1)
         ii += 1;
         s.children(ii) = hdl2struct (cb);
       elseif (ncb > 1)
         error ("hdl2struct: more than one colorbar found");
